backend/utils/request/feature_flag.py

```diff
@@ -3,7 +3,6 @@
 from typing import Optional
 
 from unstract.flags.client.evaluation import EvaluationClient
-from unstract.flags.client.flipt import FliptClient
 
 
 def check_feature_flag_status(
```

Comment:
Is this required here anymore since we also moved this to the flags.

Reply:
Removed and importted from library

Reply:
:+1: